scripts/pylib/twister/twisterlib/hardwaremap.py

```diff
@@ -215,6 +221,10 @@ def load(self, map_file):
             pre_script = dut.get('pre_script')
             post_script = dut.get('post_script')
             post_flash_script = dut.get('post_flash_script')
+            flash_timeout = dut.get('flash_timeout') or self.options.device_flash_timeout
+            flash_with_test = dut.get('flash_with_test')
+            if flash_with_test is None:
+                flash_with_test = self.options.device_flash_with_test
```

Comment:
Why not using a one-liner, just like above? Am I missing some logic?
`flash_with_test= dut.get('flash_with_test') or self.options.device_flash_with_test`

I guess functionally there is no difference? If so, can you please align them? I know it is just a nit, but it took me some time to figure out if those 2 are the same logic seeing them just next to each other but with different implementation, questioning my understanding of the logic.

Reply:
@PerMac 
> Why not using a one-liner, just like above? Am I missing some logic?
 `flash_with_test= dut.get('flash_with_test') or self.options.device_flash_with_test`

in case of a bool variable this one-liner works the same for `None` and for `False`,
so with that `flash_with_test: False` in device map doesn't override `--device-flash-with-test` at the
command line.